### PR TITLE
Bump agent to eec7f7b

### PR DESCRIPTION
**Pull request by luismiramirez**
Normalize cgroups v2 CPU stats.

### CODE REVIEW

File: test/integration/diagnose
Comment (tombruijn) [inline test/integration/diagnose:1]: I think that if you revert this change the test suite will pass.

Comment (luismiramirez) [reply test/integration/diagnose:1]: argh, forgot to update the submodule on this one
